android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -343,7 +349,12 @@ public void onMdnsUrlFound(String url) {
     }
   }
 
-  /**
+  @Override
+  public void onSsdpUrlFound(String url) {
+      onMdnsUrlFound(url);
```

Comment:
For specificity, how about we put the code that is currently in onMdnsUrlFound:

    if (!mUrlToUrlMetadata.containsKey(url)) {
      mUrlToUrlMetadata.put(url, null);
      // Fabricate the adapter values so that we can show these ersatz beacons
      String mockAddress = generateMockBluetoothAddress(url.hashCode());
      int mockRssi = 0;
      int mockTxPower = 0;
      // Fetch the metadata for the given url
      MetadataResolver.findUrlMetadata(getActivity(), NearbyBeaconsFragment.this, url, mockTxPower, mockRssi);
      // Update the ranging info
      mNearbyDeviceAdapter.updateItem(url, mockAddress, mockRssi, mockTxPower);
      // Force the device to be added to the listview (since it has no metadata)
      mNearbyDeviceAdapter.addItem(url, mockAddress, mockTxPower);
    }

 into another method called something like onLanUrlFound or onNonBleUrlFound. Then both on onMdnsUrlFound and onSsdpUrlFound will call that new method?